.github/workflows/publish_master.yml

```diff
@@ -8,21 +8,25 @@ jobs:
   analyze:
     runs-on: ubuntu-latest
     name: Dart Analyze
-
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: subosito/flutter-action@v2
+        with:
+          flutter-version: '2.x'
+          channel: 'any'
       - run: flutter pub get
       - run: flutter analyze
-  score:
-    runs-on: ubuntu-latest
-    name: Dart package score
 
-    steps:
-      - uses: actions/checkout@v2
-      - uses: axel-op/dart-package-analyzer@v3
-        with:
-          githubToken: ${{ secrets.GITHUB_TOKEN }}
+  # TODO enable this after upgrading to dart 3.x
```

Comment:
Is the upgrade planned soon?

Reply:
Not, really no

Reply:
I'd prefer to remove the code that's commented out, create an issue for the upgrade, and reference the created issue in the TODO.

Reply:
Yes, good shout